docs/changes/modeling/11978.api.rst

```diff
@@ -0,0 +1,7 @@
+Removed the following deprecated modeling features:
+    ``~astropy.modeling.utils.ExpressionTree`` class,
```

Comment:
extreme minor nit-pick: remove the `~` as these are not rendered to sphinx links. otoh I think it's good to keep their full API path in the changelog, even when they may be available from a different namespace, too.

Reply:
See ed99fdd